src/Arcus.Templates.Tests.Integration/Worker/ServiceBusWorkerProject.cs

```diff
@@ -19,12 +19,12 @@ namespace Arcus.Templates.Tests.Integration.Worker
     /// <summary>
     /// Project template to create Azure ServiceBus Queue worker projects.
     /// </summary>
-    public class ServiceBusQueueWorkerProject : TemplateProject, IAsyncDisposable
+    public class ServiceBusWorkerProject : TemplateProject, IAsyncDisposable
```

Comment:
@tomkerkhove not sure what would be best. Should I create two child classes or should I place it here in the same class and provide diff methods.

Reply:
This is just for testing so would keep it as is, no?

Reply:
Yeah, testing.
It maybe more 'reasonable' to have sep classes for sep project templates. IDK, that's why I asked 😄 .